config/secrets/test/jwt_private.pem

```diff
@@ -0,0 +1,54 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----
```

Comment:
this woud be my idea:
https://github.com/oat-sa/simple-roster/pull/176/files#r540907877

Reply:
Sorry, but this won't change. Because of the concern that different functionality even that may be using "same" keys, should be separated fully. And, especially, feature of the app vs "feature" of the dependency. I guess, noone wants to solve problems later if for some reason smth would change in lti1p3 library and we would be in situation when we are dependent from them and need to either re-generate keys or whatsoever. Better to have it separated.
p.s.
I'll leave this conversation opened

Reply:
IMO the security keys should be part of the application not part of each feature. So according to what you are saying every time that we create a new feature which requires security layer with key pairs we will need to create new key pairs. For me this I very bad for multiple reasons. Just look our environment vars they're growing a lot because of these kind of decisions. Take a look into other projects on the GitHub for example. You will see that most of the time the key pair belongs to the application not the feature. :(